### DISCUSSION TITLE
Window size

### DISCUSSION THREAD

**Discussion by filipmagula** [anonieX/alienworldsauto#3] [Ideas]
Hi anonieX, 

at first - good job. It works pretty well. 
I had two issues when trying to run the script:
1) Devtools (F12) needs to be opened to force selenium to start the sequence - not sure why, maybe just an issue on my side. 
Working fix: option.add_argument('auto-open-devtools-for-tabs')

2) The window size used by Brave on startup will be probably different for each PC, therefore it needs to be set for specific size. 
Partial fix: I have found that 925x982 works quite fine. "option.add_argument('window-size=...')" could be partial workaround, but needs some tweaking due to panels, adress bar, etc.

F.

**Comment by anonieX** [comment]
1. Probably an issue on your side. Works on my machine without devtools open
2. Thanks a lot for this, uploading a new version with this fix right now

**Comment by anonieX** [comment]
Hey, this was all fixed in 1.1.2, please try it and let me know if it works for you. Thanks

https://github.com/anonieX/alienworldsauto/releases/tag/1.1.2
